File: RedisAPI/Client/Korisnik.js
export class Korisnik {
    constructor(id, Prezime, Email, Adressa, Ime, Userime, jwt) {
        this.id = id;
        this.prezime = Prezime;
        this.email = Email;
        this.adressa = Adressa;
        this.ime = Ime;
        this.userime = Userime;
        this.jwt = jwt;
    }
}

File: RedisAPI/Models/Korisnik.cs
using Redis.OM.Modeling;

namespace Redis.OM.Skeleton.Model;


[Document(StorageType = StorageType.Json, Prefixes = new []{"Korisnik"})]
public class Korisnik
{
    [RedisIdField] [Indexed]public string Id { get; set; }

    [Indexed] public string Prezime {get; set;} = String.Empty;

    [Indexed] public string Email {get; set;} = String.Empty;

    [Indexed] public string Adressa {get; set;} = String.Empty;

    [Indexed] public string BrTelefona {get; set;} = String.Empty;

    [Indexed] public string Ime {get; set;} = String.Empty;

    [Indexed] public string Userime {get; set;} = String.Empty;

    [Indexed] public string Passwordhash {get; set;} = String.Empty;

    [Indexed] public string Passwordsalt {get; set;} = String.Empty;
    

    //nacin placanja
}